usegalaxy.org/convert_formats.yml

```diff
@@ -42,3 +42,5 @@ tools:
   owner: galaxyp
 - name: mz_to_sqlite
   owner: galaxyp
+- name: gffread
+  owner: devteam
```

Comment:
I think that the current version is hosted in the IUC repo (instead of devteam).

Reply:
It is, but the owner in the TS is still `devteam`.